openwisp_notifications/tests/test_notifications.py

```diff
@@ -323,6 +322,35 @@ def test_default_notification_type(self):
         )
         self.assertEqual(n.email_subject, '[example.com] Default Notification Subject')
 
+    def test_generic_notification_type(self):
+        self.notification_options.pop('verb')
+        self.notification_options.update(
+            {
+                'message': '[{notification.actor}]({notification.actor_link})',
+                'type': 'generic_message',
+                'description': '[{notification.actor}]({notification.actor_link})',
+            }
+        )
```

Comment:
I improved this test to verity markdown rendering of message and description works.